### PR TITLE
feat: add confirm dialog for delete credentials

### LINKED ISSUE #1511: Show the confirm dialog when you delete the credentials.
### Main idea

When the user clicks the delete button, the confirm dialog appears in most cases. But not for credentials.
Let's provide a confirm dialog to the deleting credential too.

### Alternative ideas

_No response_

### Anything else?

you can test in the credential tab of the users(credential) page.
the delete button is like below.
image

### PR DESCRIPTION
**Pull request by raipen**
Resolve #1511.

### CODE REVIEW

File: src/components/backend-ai-credential-list.ts [hunk 1]
```diff
@@ -396,19 +399,33 @@ export default class BackendAICredentialList extends BackendAIPage {
...
    */
   _deleteKey(e) {
-    const controls = e.target.closest('#controls');
-    const accessKey = controls['access-key'];
-    globalThis.backendaiclient.keypair.delete(accessKey).then((response) => {
+    globalThis.backendaiclient.keypair.delete(this.deleteKeyPairAccessKey).then((response) => {
       if (response.delete_keypair && !response.delete_keypair.ok) {
         throw new UnableToDeleteKeypairException(response.delete_keypair.msg);
       }
+      this.notification.text = PainKiller.relieve('Deleted the access key successfully.');
```
Comment (inureyes) [inline src/components/backend-ai-credential-list.ts]: It should be i18n text.

**Review by inureyes — CHANGES_REQUESTED** [review]
Please add i18n resource to notification.

File: src/components/backend-ai-credential-list.ts (hunk 1 above)
Comment (inureyes) [reply src/components/backend-ai-credential-list.ts]: Please refere `_text()` function.

Comment (raipen) [reply src/components/backend-ai-credential-list.ts]: Fixed

**Pull request merged**